app/views/posts/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Here is a selected post</h1>
+<% if @user.user_recent_posts.empty? %>
+  <h2>No Posts</h2>
+  <p>Kindly submit a post</p>
+<% else %>
+  <% @user.user_recent_posts.each do |post| %>
+    <%= link_to user_post_path(user_id: @user.id, author_id: post.id) do %>
+      <div>
+        <div>
+          <h1><%= post.title %></h1>
+          <p><%= post.text %></p>
+        </div>
+        <div>
+          <p>Comments: <%= post.comment_counter %>, Likes: <%= post.likes_counter %></p>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+  <div>
+    <%= link_to 'See all posts', user_posts_path(user_id: @user.id) %>
+  </div>
+<% end %>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the post-show page 
Take a look at what the wireframe looks like for the post-show page. Kindly implement this.
<img width="496" alt="Screenshot 2023-06-17 at 20 16 48" src="https://github.com/Lornakaboro/Rails-blog/assets/58692590/274380ca-dd65-428d-b47c-1a84756d39f5">